### PR TITLE
Add wiki: rocksdb-in-cita.

### PR DESCRIPTION
**Pull request by leeyr338**
Add wiki: rocksdb-in-cita

### CODE REVIEW

File: docs/wiki/rocksdb-in-cita.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+## RocksDB 简单介绍
+当前 CITA 使用 RocksDB 作为其底层存储的数据库。
+RocksBD 采用的是 LSM-Tree(Log-Structured-Merge-Tree), 如下图所示：
+
+<img src = "https://github.com/cryptape/cita/blob/develop/docs/wiki/images/rocksdb-lsm-tree.png?raw=true" width="50%" height="50%">
```
Comment (kaikai1024) [inline docs/wiki/rocksdb-in-cita.md]: Does `LSM-Tree`  work?

Comment (leeyr338) [reply docs/wiki/rocksdb-in-cita.md]: I have tried some relative links in citahub-doc, and it doesn't work.

Comment (kaikai1024) [reply docs/wiki/rocksdb-in-cita.md]: Not citahub-doc, just check if github can show the pic.

**Comment by kaikai1024** [comment]
The wiki or this doc, we only need to keep one and use link at other place.
we can keep this and the wiki use the link.

File: docs/wiki/rocksdb-in-cita.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+LSM 树的设计思想非常简单，它的原理是把一颗大树拆分成N棵小树，它首先写入到内存中（内存没有寻道速度的问题，随机写的性能得到大幅提升），
```
Comment (ouwenkg) [inline docs/wiki/rocksdb-in-cita.md]: N, 左右两边空格

Comment (ouwenkg) [reply docs/wiki/rocksdb-in-cita.md]: 有个地方不太理解, 看这个图来理解的话, LSM 的设计思想, 好像是把多棵小树, 合并成为一个大树. 或者是我哪个地方没有理解对?

Comment (leeyr338) [reply docs/wiki/rocksdb-in-cita.md]: 这是相对于其它系统讲的，指的是其它系统只有一个树。

**Pull request merged**